sim/src/main/scala/midasexamples/AutoCounterModule.scala

```diff
@@ -71,6 +71,15 @@ class AutoCounterModuleChild extends MultiIOModule {
   io.oddlfsr := odd_lfsr
 }
 
+/** Demonstrate explicit instrumentation of AutoCounters via PerfCounter 
+ *
+ * Toplevel Chisel class suitable for use as a GoldenGate 'target' as described by the docs
+ *
+ * @see https://docs.fires.im/en/latest/Advanced-Usage/Debugging-and-Profiling-on-FPGA/AutoCounter.html#ad-hoc-performance-counters
```

Comment:
Is there a better way to reference explicit links into the docs that is more maintainable?

Reply:
that's a good question. I'm not sure. There's both the brittleness of using a URL and the versioning problem we have to contend with here.